client/templates/nav/nav.js

```diff
@@ -40,15 +40,11 @@ Template.notificationsDropdown.onRendered(function() {
 
 Template.notificationsDropdown.helpers({
   hasNotifications: function(){
-    if(Meteor.user() && Meteor.user().notificationsCount() > 0){
-      return true;
-    }
-
-    return false;
+    return Meteor.user() && Meteor.user().notificationsCount() > 0;
```

Comment:
Shouldn't this change? We aren't storing the `notificationsCount` method on the user object anymore are we?